### PR TITLE
fixing CDT

### PR DESCRIPTION
**Pull request by DarrenCattle**
temp FIX real fix COMING

### CODE REVIEW

**Comment by DarrenCattle** [comment]
Austin Pray: lol sup

File: scripts/kkreds.coffee [hunk 1]
```diff
@@ -11,6 +11,8 @@
 #
 # Author:
 #   austinpray
+# Co-arthur:
```
Comment (austinpray) [inline scripts/kkreds.coffee:14]: are you dumb?

image

File: scripts/kkreds.coffee [hunk 2]
```diff
@@ -77,7 +79,8 @@ module.exports = (robot) ->
   triggers = [
     "(gibbe|give( me)?) money",
     "pay me( bitch)?",
-    ":watermelon:"
+    ":watermelon:",
+    "d is god"
```
Comment (austinpray) [inline scripts/kkreds.coffee]: pls remove

**Comment by ridhoq** [comment]
dank Arthur memes incoming

**Pull request closed without merging**